scripts/base/init-bare.zeek

```diff
@@ -5581,6 +5581,23 @@ const digest_salt = "Please change this value." &redef;
 ## :zeek:see:`find_all_ordered` BIFs.
 const max_find_all_string_length: int = 10000 &redef;
 
+## How many rounds to go without checking IO sources with file descriptors
+## for readiness by default. This is used when reading from traces.
+##
+## Very roughly, when reading from a pcap, setting this to 100 results in
+## 100 packets being processed without checking FD based IO sources.
+const io_poll_interval_default = 100 &redef;
```

Comment:
You should add some comments to these about how they generally shouldn't be changed in practice.

Reply:
I added a `.. note::` about that, thanks.